### DISCUSSION TITLE
Disk formatted and no backup, data recovery, Linux Ubuntu

### DISCUSSION THREAD

**Discussion by Clahro** [Authenticator-Extension/Authenticator#839] [General]
Hi,

the other day I formatted the partition on which the Chrome extension was installed. I haven't done a backup and I need to gain access to the account.

I was thinking of trying to recover the files that save all the account details needed and to copy it to the new system. 

The file fomat of the disk was ext3 and Linux mint 18.2 was installed.

Now, I wonder which files would I have to recover and in which folder I would have to copy it on the new system. Do you have any recoomendation for which software to use to recover those files? Btw. I have installed Windows on another partition so I could use a windows software if that is of any help.

Help is much appreciated,
Clahro

**Comment by mymindstorm** [comment]
Data is stored in your browser data directory. For chrome you would need to restore: https://chromium.googlesource.com/chromium/src/+/HEAD/docs/user_data_dir.md#Linux
